neqo-http3/src/connection_client.rs

```diff
@@ -86,7 +81,7 @@ impl Http3Client {
         local_addr: SocketAddr,
         remote_addr: SocketAddr,
         conn_params: ConnectionParameters,
-        http3_parameters: &Http3Parameters,
+        http3_parameters: Http3Parameters,
```

Comment:
Any sense in having Http3Parameters include ConnectionParameters?  You could make it impl DerefMut<Target=ConnectionParameters> so that you can use the same interface to manage everything.